src/models/ImageBlock.js

```diff
@@ -0,0 +1,16 @@
+import RelatedImage from "./RelatedImage";
+
+export default class ImageBlock {
+    constructor(centralId, oldCentralImage, centralImageTopPosition, centralImageLeftPosition, imagePositions, relatedImages) {
```

Comment:
probably better to change `centralImageTopPosition, centralImageLeftPosition` to centralImagePosition with an object with  { top, left }

why `oldCentralImage` and not `centralImage` ?

Reply:
Yes, the object matches with the approach used in the template and this solution is better.

Regarding the oldCentralImage, it is because that represents the old central image and not the current one. The reason behind that is because we have 2 blocks and when a block is removed, the previous central image is removed too. But we need it to complete the previous block. For this reason, I keep track on this image to display it when the block is removed.